core/templates/pages/preferences-page/preferences-page.component.ts

```diff
@@ -245,8 +245,8 @@ export class PreferencesPageComponent {
       this.loaderService.hideLoadingScreen();
     });
 
-    this.userCanDeleteAccount = AppConstants.ENABLE_ACCOUNT_DELETION;
-    this.userCanExportAccount = AppConstants.ENABLE_ACCOUNT_EXPORT;
+    this.userCanDeleteAccount = true;
+    this.userCanExportAccount = true;
```

Comment:
Remove these variables from the codebase too.

Reply:
Done